bower.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "bootstrap-winjs",
-  "version": "0.2.0",
+  "name": "WinStrap",
```

Comment:
Good catch Martin. The intracaps were updated in #38. Thinking I'll merge that into this branch and then close that PR. That way we get everything when we take this one.